public/js/dashboard.js

```diff
@@ -263,6 +289,22 @@ app.controller('lowerThirdsCGController', ['$scope', 'localStorageService', 'soc
 
         var stored = localStorageService.get('lower_thirds');
 
+        const showLiveLT = $scope => {
```

Comment:
Expanding `showLiveLT` to `showLiveLowerThird` etc helps with readability of code.

Reply:
argeed will fix in a mo